docs/en/aws-elastic-serverless-forwarder.asciidoc

```diff
@@ -79,6 +79,26 @@ The forwarder can ingest logs contained within the payload of an Amazon SQS body
 
 You can set up a separate SQS queue for each type of log. The config parameter for {es} output `es_datastream_name` is mandatory. If this value is set to an {es} data stream, the type of log must be correctly defined with configuration parameters. A single configuration file can have many input sections, pointing to different SQS queues that match specific log types.
 
+[discrete]
+[[at-least-once-delivery]]
+== At-least-once delivery
```

Comment:
I think this section might be easier to digest if we split it in two, providing details for the two queues in two separate sections/sub-sections (e.g "More details about the Replay queue" or "Replay queue internals).
Each section can follow a pattern like the following:
- What part this queue plays in ensuring at-least-once delivery
- When ESF sends messages to this queue
- Failures/DLQ scenarios

I would also add a note about the recent changes we made, removing `sqs.deleteMessage` calls and their impact.